packages/libassuan/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://www.gnupg.org/related_software/libassuan/
 TERMUX_PKG_DESCRIPTION="Library implementing the Assuan IPC protocol used between most newer GnuPG components"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="2.5.7"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="3.0.1"
 TERMUX_PKG_SRCURL=https://www.gnupg.org/ftp/gcrypt/libassuan/libassuan-${TERMUX_PKG_VERSION}.tar.bz2
-TERMUX_PKG_SHA256=0103081ffc27838a2e50479153ca105e873d3d65d8a9593282e9c94c7e6afb76
+TERMUX_PKG_SHA256=c8f0f42e6103dea4b1a6a483cb556654e97302c7465308f58363778f95f194b1
 TERMUX_PKG_DEPENDS="libgpg-error"
 TERMUX_PKG_BREAKS="libassuan-dev"
 TERMUX_PKG_REPLACES="libassuan-dev"
+TERMUX_PKG_BREAKS="gnupg (<< 2.4.5-3), gpgv (<< 2.4.5-3), gpgme (2.4.5-3), pinentry (<< 1.3.1-1), pinentry-gtk (<< 1.3.1), profanity (<< 0.14.0-3)"
```

Comment:
Should be on the same line with the other BREAKS. No operator for gpgme?

Reply:
Sorry, missed this one. Done.